curriculum/challenges/english/blocks/lab-symmetric-difference/68ad9821ee41baad9cb0fd4e.md

```diff
@@ -67,6 +67,11 @@ assert.deepEqual(diffArray(
   ["andesite", "grass", "dirt", "dead shrub"],
   ["andesite", "grass", "dirt", "dead shrub"]
 ), []);
+
+assert.deepEqual(diffArray(
+  ["diamond", "stick", "apple"],
+  ["stick", "emerald", "bread"]
+), ["diamond", "apple", "emerald", "bread"]);
 ```
```

Comment:
please add the requested five tests in the same format as the others, with its own hint text, not as a hidden test